packages/techdocs-common/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # @backstage/techdocs-common
 
+## 0.3.2
+
+### Patch Changes
+
+- 7ec525481: 1. Added option to use AWS S3 as a choice to store the static generated files for TechDocs.
+- f8ba88ded: Fix for `integration.github.apiBaseUrl` configuration not properly overriding apiBaseUrl used by techdocs
+- 00042e73c: Moving the Git actions to isomorphic-git instead of the node binding version of nodegit
+- Updated dependencies [5ecd50f8a]
+- Updated dependencies [00042e73c]
+- Updated dependencies [0829ff126]
+- Updated dependencies [036a84373]
+  - @backstage/backend-common@0.4.2
+  - @backstage/integration@0.1.5
+
 ## 0.3.1
 
```

Comment:
Confused about some of the changes in this PR like this and deletion of changesets that do not exist, etc. Could this use a rebase with the latest master brach?

Reply:
I think that's because this branch started when they exist and then I merged master in when the change to remove the old changesets came in.

Those files don't exist on master to it's safe to merge